README.rst

```diff
@@ -32,6 +32,14 @@ to support modern computational graph libraries such as PyTorch and
 TensorFlow in order to make use of features such as autodifferentiation
 and GPU acceleration.
 
+User Guide
+----------
+
+For an in depth walkthrough of usage of the latest release of ``pyhf`` visit the |pyhf tutorial|_.
```

Comment:
I'm trying to sync language here

```console
$ git grep 'in depth walkthrough'
.github/ISSUE_TEMPLATE/config.yml:    The pyhf tutorial is continually updated and provides an in depth walkthrough
README.rst:For an in depth walkthrough of usage of the latest release of ``pyhf`` visit the |pyhf tutorial|_.
```

but happy to change it.

Reply:
looks fine to me.